content/blog/2023/11/10/2023-11-10-jenkins-october-newsletter.adoc

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: "Jenkins October 2023 Newsletter"
+tags:
+- jenkins
+- newsletter
+- community
+- contribute
+authors:
+- dduportal
+- markewaite
+- gounthar
+- wadeck
+- kmartens27
+- alyssat
+opengraph:
+  image: /images/post-images/2023/02/07/2023-02-07-jenkins-newsletter/centered-newsletter.png
+links:
+discourse: true
+---
+
+image:/images/post-images/2023/02/07/2023-02-07-jenkins-newsletter/centered-newsletter.png[Jenkins September Newsletter]
+
+== Key Takeaways
+
+* JDK21 is available on the infrastructure and in official Docker images too.
+* 💥Breaking change: set Java 17 as default for LTS.
+* Prototype has been removed as of weekly 2.426
+
+
+[[security-fixes]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/security.png[Security Update]
+
+Contributed by: author:wadeck[Wadeck Follonier]
+
+* Core security advisory published on October 18
+** link:/security/advisory/2023-10-18/[https://jenkins.io/security/advisory/2023-10-18/]
+** Includes an essential Jetty update that provided multiple fixes.
+* Plugin security advisory published on October 25
+** link:/security/advisory/2023-10-25/[https://jenkins.io/security/advisory/2023-10-25/]
+** Multiple high score vulnerabilities in various plugins
+* During Hacktoberfest, the Content Security Policy topic got some attention with several PRs being proposed.
+** For more information, refer to the https://issues.jenkins.io/browse/JENKINS-60865[related ticket].
+
+[[Governance]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+The link:/blog/2023/11/06/introducing-2-2-2-java-support-plan/[2 + 2 + 2 Java support plan] has been announced as the Jenkins project approach to support Java versions now and in the future.
+We plan to generally support two Java LTS releases at any one time, with a transition period over the next 2 years.
+
+Java 21 support by the Jenkins project is looking very, very good.
+Jenkins 2.426.1 is scheduled to be released Nov 15, 2023 with full support for Java 21.
+Read more about the Java support plan in the link:/blog/2023/11/06/introducing-2-2-2-java-support-plan/[recent blog post from author:basil[Basil Crow] ].
```

Comment:
Nested hyperlink won't have the same result as two separate hyperlinks.
```suggestion
Read more about the Java support plan in the link:/blog/2023/11/06/introducing-2-2-2-java-support-plan/[recent blog post] from author:basil[Basil Crow].
```

Reply:
Thanks @MarkEWaite